scripts/adr-doctor.sh

```
@@ -0,0 +1,242 @@
+  if ! shellcheck "$HOOK" >/dev/null 2>&1; then
+    check_fail "$HOOK: shellcheck failed"
+  fi
+fi
+
+# --- forbid_skill_commits (INV-A2) ---
+note "forbid_skill_commits (INV-A2)"
+SKILL="$REPO_ROOT/.claude/skills/capture-adrs/SKILL.md"
+if [ -f "$SKILL" ]; then
+  # Look for command-shaped forbidden strings (in code blocks or inline-command form).
+  # The Anti-patterns section may mention them in 'DO NOT commit' prose; allow that.
+  if grep -qE '(^\s*\$\s*(jj commit|jj describe|git commit|git add)|^\s*`(jj commit|jj describe|git commit|git add)`)' "$SKILL"; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)